pretalx_public_voting/forms.py

```diff
@@ -169,7 +173,9 @@ class Meta:
             "start": HtmlDateTimeInput,
             "end": HtmlDateTimeInput,
             "limit_tracks": EnhancedSelectMultiple(color_field="color"),
+            "limit_submission_types": EnhancedSelectMultiple(color_field="color"),
```

Comment:
Submission types don't have colours, so please remove the `color_field` argument here.